### DISCUSSION TITLE
[Feature request] Implement gravity override for all bodies and not just vehicle constraints

### DISCUSSION THREAD

**Discussion by PiMoNFeeD** [jrouwe/JoltPhysics#1231] [Q&A]
In my project, I want to be able to override gravity on per-object basis, and while I could mess with gravity factor, that won't help when my global gravity shifts if I want my overridden gravity to stay the same. That would also help make vehicle constraint implementation cleaner, as that won't need to do the hack of setting vehicle gravity factor to 0 and instead applying it manually.

**Comment by jrouwe** [comment]
Hello,

I'm currently not planning to create a property to do this since it is really easy to implement per object custom gravity yourself, see:

https://github.com/jrouwe/JoltPhysics/blob/1bc02f61c53bf178a4715f0c41aca44b892b3a84/Samples/Tests/Character/CharacterPlanetTest.cpp#L173-L198

**Reply by PiMoNFeeD** [reply]
That would require me to set gravity factor to 0 so it doesn't apply it on it's own, right?

**Reply by jrouwe** [reply]
Correct
